qml/Ping360Visualizer.qml

```diff
@@ -71,6 +71,51 @@ Item {
                     }
                 }
 
+                // Spinner that shows the head angle
+                Shape {
```

Comment:
It is not immediately clear what each child's purpose is. A brief comment will help :)

Reply:
done.